src/guide/jbs-jdk-bug-system.md

```diff
@@ -431,10 +431,10 @@ This table contains some frequently used JBS labels and their meaning. Please he
                            [*(Rel)*[-fix-yes]{.jbs-label}]{#rel-fix-yes}<br />
                            [*(Rel)*[-fix-no]{.jbs-label}]{#rel-fix-no}</td>
     <td class="dictionary">
-      Used in ramp down phase 2 to indicate that an issue would be of interest to get integrated into release *(Rel)*. E.g., [jdk12u-fix-request]{.jbs-label}
+      *(Rel)*[-fix-request]{.jbs-label} is used in ramp down phase 2 to indicate that an issue would be of interest to be integrated into release *(Rel)*. E.g., [jdk12-fix-request]{.jbs-label}
 
-      *(Rel)*[-fix-SQE-ok]{.jbs-label} is used to indicate that the issue will be covered by the test plan for *(Rel)*. E.g., [jdk12u-fix-SQE-ok]{.jbs-label}<br />
-      *(Rel)*[-fix-yes]{.jbs-label} and *(Rel)*[-fix-no]{.jbs-label} are used to indicate wether an issue has been approved for backport to *(Rel)*. E.g., [jdk12u-fix-yes]{.jbs-label}
+      *(Rel)*[-fix-SQE-ok]{.jbs-label} is used to indicate that the issue will be covered by the test plan for *(Rel)*. E.g., [jdk12-fix-SQE-ok]{.jbs-label}<br />
+      *(Rel)*[-fix-yes]{.jbs-label} and *(Rel)*[-fix-no]{.jbs-label} are used to indicate wether an issue has been approved for backport to *(Rel)*. E.g., [jdk12-fix-yes]{.jbs-label}
```

Comment:
Yet `jdk12-fix-yes` isn't applicable to backports, it's for post RDP2 integrations (which are also *backports* since the initial fix goes to mainline which is at *Rel+1*). Maybe a different wording should be used to clarify…

Reply:
As you say, I think this part would benefit from clarifying what goes for the updates and for mainline.